Functions/xASL_bids_CompareStructures.m

```diff
@@ -136,15 +136,19 @@
 function reportTable = xASL_bids_CompareStructures_AddEntriesToTable(reportTable,results,datasetA,datasetB)
 
     % Go through missing folders
-    if size(results.(datasetA).missingFolders,1)>0 && ~isempty(results.(datasetA).missingFolders{1})
+    if size(results.(datasetA).missingFolders,1)>0 && ...
```

Comment:
Not really wrong, but would it not be easier to use numel rather than size 1 and size 2?
This applies also to lines 139-169. Including the for statements etc.

Reply:
True, forgot that numel is easier here.

Reply:
I'm not 100% sure about the for loops, but for the rest it's done cd90e1b and tested...

```
UnitTest = xASL_ut_function_xASL_bids_CompareStructures(TestRepository);

>> UnitTest

UnitTest = 

  struct with fields:

     tests: [1×4 struct]
    passed: 1
```